examples/asr/asr_chunked_inference/rnnt/speech_to_text_buffered_infer_rnnt.py

```diff
@@ -135,7 +136,9 @@ class TranscriptionConfig:
     stateful_decoding: bool = False  # Whether to perform stateful decoding
 
     # Merge algorithm for transducers
-    merge_algo: Optional[str] = 'middle'  # choices=['middle', 'lcs'], choice of algorithm to apply during inference.
+    merge_algo: Optional[str] = (
```

Comment:
Suggestion: set `merge_algo` to `None` (meaning "auto")
In the code use something like: 
```python
if merge_algo is None:
    merge_algo = "tdt" if model.is_tdt() else "middle"
```

Reply:
Good idea.

Reply:
Thanks! Can you also set default to `None` in the config?

Reply:
Done.